src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -26,12 +28,18 @@ protected override SearchView CreatePlatformView()
 
 		protected override void ConnectHandler(SearchView platformView)
 		{
+			FocusListener.Handler = this;
+			platformView.SetOnQueryTextFocusChangeListener(FocusListener);
```

Comment:
What if we subscribe to FocusChanged on the EditText instead of adding a listener? From the source code it looks like all android does is adds a focus listener to the edittext and then forwards the call to this listener

https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/widget/SearchView.java;l=345;drc=20ffbd23f13b00dd32f6cc4c7cfed2cc8ea46614?q=setOnQueryTextFocusChangeListener